predict.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
+import time
+import numpy as np
+from termcolor import colored
+import pickle
+import common.py
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--test_dataset_size", default=10000, type=int, dest="test_dataset_size")
```

Comment:
Make default 100 million, I believe most often you'll just want to predict for the whole test file.